templates/vhost.j2

```diff
@@ -0,0 +1,32 @@
+server {
+    listen {{ item.listen | default('80 default_server') }};
```

Comment:
I think when I originally built the role I was only thinking of one server block, and if someone didn't set `listen` then that should default to `default_server`.

Technically the first/primary server should be set to `default_server` (at least imho), but since this is meant to be a generic server template at this point, I'd rather we just set it to `default('80')`.